### ISSUE TITLE
North Atlantic dense water formation

### ISSUE THREAD

**Issue by adele-morrison** [willaguiar/DSW-collaborative-project#32]
Bit of a left field idea: is it worth checking how North Atlantic dense water formation responds to changes in vertical resolution? 

i.e. for an initial look, plot zonal average overturning in depth or density coordinates in the Atlantic.

**Comment by PaulSpence** [comment]
Definitely a good idea. We have ignored the NAtl overflows for too long!

**Comment by willaguiar** [comment]
Ok, I did a quick check of the Overturning in the last year of each simulation, to see how the N.A .respond to thickening the top cell.
Figures are...

Overturning contours: 
Top: Overturning in the last year of control (1 meter top cell)
Middle: Anomaly (5mtop - control) on colors, lines are the same as top plot for the control
Bottom: Anomaly (halfmtop - control) on colors, lines are the same as top plot for the control

Line plot:
[a] maximum overturning at 26N, for $\sigma_2>1036$, i.e., NADW flow
[b] Absolute minimum overturning at 60S, for $\sigma_2>1037.05$, i.e., NADW flow

**Interestingly it seems that the effect of _dz_ in the North Atlantic overturning is reversed, i.e., as we increase the thickness of the top cell, we end up having more NADW, and less AABW**
Any ideas on what could be the difference here? 

Next steps is to analyze MLD in the North Atlantic and SWMT to see how/where NADW is formed in the model. Might give us a hint on what to look for.


possible hypothesis: DSW formation in the South is driven by salt fluxes, while in North Atlantic deep convection is likely driven by heat fluxes. So perhaps the sensitivity of the top cell depends on what is driving the mass transformation at surface?

![Overturning_changes](https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/9a1763a9-3834-4d44-9740-f636478e0464)
![Overturning_indexes](https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/5f743fa2-1659-4934-bdb9-55e5414fa31b)

**Comment by wghuneke** [comment]
Thanks for looking into this. Very intriguing that the result is opposite in the NA - I agree that checking whether salinity vs temperature is dominating the convection might help understanding what is going on.

Just noting here that the observed AMOC at 26N is about 17 Sv (McCarthy et al., 2015, 2004-2012 average) and the values here are much higher - possibly not the exact same calculation. Anyway, just saying that it might be the case that in the NH, the model produces more deep convection with a thicker uppermost grid cell and thereby overshoots. So the result would be consistent with our Antarctic analysis, i.e. a thinner uppermost grid cell is desirable.

**Comment by willaguiar** [comment]
Good Point Wilma..... That overturning line-plot is measured for the maximum below $\sigma_2>1036$, but that is not really the centre of the NADW overturning cell (which is actually at $\sigma_2>1036.5$), on the b contour. When I do the same line plot for  $\sigma_2>1036.5$ indeed we get ~17SV for halfmtop, ~19SV for control by the end of the last year (Which supports your idea)

**Comment by dkhutch** [comment]
Yes very interesting that it's like a bipolar seesaw type response. Is it
just good luck that we get closer to observations this way? In my mind 19
Sv in the model is a respectable answer for Amoc and a bit of ad hoc tuning
could change it either way.

It's interesting to see this though. Yes would be great to get some idea of
whether temp or salt fluxes are more in control in each location.

I think we saw on Friday that the expansion of MLD regions in the Antarctic
margins tended to be on the periphery of the existing Maxima locations.
Whereas when it contracted, it tended to contract in the middle of the
Maxima, right? Wonder if it's similar pattern in the north.

**Comment by willaguiar** [comment]
here it goes some considerations of the NADW formation on the North Atlantic (SWMT and Mass transports).
 _Apologies in advance for the loads of plots_

Based on winter Mixed Layer Depth (DJFMA, to pinpoint deep convection), I defined the following regions and boundaries for SWMT and to calculate the mass transports.
<img width="604" alt="Screen Shot 2023-12-13 at 11 50 11 AM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/4c936c7f-649b-49ef-9ad2-e15f54da23f3">
**SWMT and Mass flow in the Nordic Seas:**
For reference, NADW seem to have $\sigma_0$ approx $27.8$, and $\sigma_2>36.7$,  in the model. 
Nordic Seas dense-SWMT seem to increase in $\sigma_0>28$, around the densest layers ([a-c]). In turn the Southward transport across the Nordic seas array increases in the densest classes ($\sigma_2>36.8$), but is a very small increase (I guess it makes sense, as these denser waters would struggle to cross the Iceland ridge)
<img width="919" alt="Screen Shot 2023-12-13 at 12 00 07 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/8e7e4223-0da4-4667-bb21-444a153cb35c">
**SWMT and Mass flow in the Northeast Atlantic:**
Here the SWMT creates waters that are too light to be NADW ( dense transformation is in $\sigma_0<27.8$). We also don’t see significant changes in SWMT between simulations. So likely no significant changes in NADW formation happen here
<img width="926" alt="Screen Shot 2023-12-13 at 12 08 08 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/00531e70-2f29-4533-91eb-a5a28abaed37">
**SWMT and Mass flow in the Labrador:**
Here there is a huge dense-SWMT occurring ( more than 30 SV), so likely the bulk of the NADW formation in the model happens here [a]. This is very common in ocean models as far as I know, but it doesn’t match much observations e.g., [OSNAP shows stronger MOC flows on East than west]( https://www.o-snap.org/).
When thickening the top cell, it seems that dense-SWMT increases for $\sigma_0>27.86$, and decreases for $\sigma_0<27.86$. So the results is not as simple as we saw for the [Antarctic Shelf SWMT]( https://github.com/willaguiar/DSW-collaborative-project/issues/20#issuecomment-1833042547) where thickening the top cell weakens the SWMT mostly in the densest water classes.
<img width="934" alt="Screen Shot 2023-12-13 at 12 17 07 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/3a1cca61-9cb7-4fcd-9baf-5b6b2d41468f">
In maps with mean of the SWMT for $\sigma_0>27.86$ in the simulations (a-f), and of anomalies in MLD (g), we can see that the region where we have increased dense transformation in the Labrador sea approximately matches where we have increased deep convection (a-b, mld in g). And that this is mostly driven by heat fluxes (c-d, in contrast with e-f, _scale in e-f is 15% of c-d_). So increasing the top cell thickness, increases and concentrates the heat-flux induced convection in that red blob in the Labrador sea, increasing the formation of the densest NADW.
<img width="906" alt="Screen Shot 2023-12-14 at 12 03 11 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/9bf566cc-9f1c-4fc6-9708-7e2bba067869">
Finally, the plot below is an Atlantic-only sum of the northward transports,. Southward NADW transport is negative in green in b ($\sigma_2>36.7$), and in c anomalies are as 5mtop-1mtop over control transport contour lines. Yellow line 36.98. We can see a decrease in UNADW southward transport ($\sigma_2<36.98$, red), and an increase in LNADW ($\sigma_2>36.98$,blue). The sum of the transport in 45N and 26N (d), show that the total NADW transport actually doesn’t change that much (~ 1SV), but the proportion of UNADW vs LNADW does.
<img width="923" alt="Screen Shot 2023-12-14 at 12 10 52 PM" src="https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/24e2517d-da1a-4150-9eb5-923cdbee51c0">
Summary:

**Most of the NADW formation in the model happens through deep convection in the Labrador Sea. Thickening the model top cell causes stronger heat-flux-induced convection in a concentrated region in the Labrador sea, which creates denser NADW. Dense NADW ( + mixing likely?) increases NADW inflow slightly ( ~ 1 SV).**

One question that remains… why thickening the top cell causes an increase in denser water transformation in the North, and decrease in the South? Perhaps this is related to transformation in the North being heat-driven, and in the south being salt-driven.

Ps:[Van Roekel experiment]( https://github.com/willaguiar/DSW-collaborative-project/issues/29#issuecomment-1769660419) with the + kpp bias in OSBL during free convection with thick top cells uses heat fluxes to drive the convection. So perhaps the issue here is the sensitivity of the kpp to heat fluxes that is not present for salt fluxes, because salt diffusion is slower.


[Code]( https://github.com/willaguiar/DSW-collaborative-project/blob/main/JupyterCodes/NA_Mass_transport_SWMT%20budget.ipynb)

**Comment by PaulSpence** [comment]
Interesting. I think some people would be unimpressed with the lack of convection in the GIN sea.

**Comment by willaguiar** [comment]
> Interesting. I think some people would be unimpressed with the lack of WMT/convection in the GIN sea.

I agree... probably the forcing might not be the best in the North Atlantic  (perhaps not much of a model issue but more of a forcing issue??)

**Comment by wghuneke** [comment]
Great analysis. @willaguiar , did we run an experiment with different KPP parameters? Do you think it would be a helpful addition? (I forgot if we had a discussion around this already and what the arguments were for/against this experiment.)

**Comment by willaguiar** [comment]
> did we run an experiment with different KPP parameters? Do you think it would be a helpful addition?

Thanks for the suggestion! I think we gave up on the idea after having similar results for MOM5-kpp and MOM6-epbl for DSW/AABW. But it might be worth to reconsider it,  especially if we further want to explain what happens in the N.A.

**Comment by adele-morrison** [comment]
Nice to see Wilton! And curious that it’s so different to the Antarctic. Did you check that the Atlantic SWMT is heat not salt dominated?

if it is heat dominated: for a thick top cell, perhaps if it’s the top cell equilibrating with the atmosphere (i.e losing heat), can you do that more efficiently with a larger volume top cell? e.g. the model doesn’t have to wait another time step to advect more heat upwards to the top cell. Just brainstorming, not sure if that’s the correct way to think about it. @AndyHoggANU thoughts on what’s going on here?

**Comment by dkhutch** [comment]
I think what @adele-morrison suggests is certainly plausible, i.e. a thickening of the top cell could somehow enable more heat exchange, while the freshwater/salt exchange is more limited in the south. I think Wilton was saying on Friday that the northern DSW formation is heat limited.

**Comment by willaguiar** [comment]
Good tip to look at heat and Salt transformation…:
Below is the comparison of SWMT from heat and salt between models. Left are the salt, heat and total components for the control experiments (1mtop), middle are the salt components for the 1mtop and 5mtop experiments, and right are the same as middle but for heat fluxes.
Top is Antarctic shelf panan (MOM6-EPBL), middle is Antarctic Shelf OM2 (KPP), and bottom is Labrador Sea OM2.
![Salt_VS_heat_transformation_hemispheres](https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/6ff5c110-9d01-40f8-a567-bb3b2250e282)
Salt and heat transformation response to top cell thickening in Antarctic Shelf is the same (decreases transformation, a to f). Antarctic heat and salt transformation response to top cell thickening is not sensitive to the mixing scheme (a,b,c compared to d,e,f). Still, increasing the top cell thickness in Labrador Sea favor (salt and heat driven) denser deep water transformation (and hinder lighter deep water transformation). So probably it isn’t the shift from salt to heat forced SWMT that causes the differing hemispheric responses.

Perhaps the different continent geographies is causing contrasting responses instead? 

I.e.,  In the Labrador Sea convection site, thickening the top cell might lead to locations with lower heat loss (lower latitudes green blob below) to cease the deep convection. But then, because there isn’t a continent north of Labrador, the convection can still happen more at North, where heat loss would be more intense and capable of surpassing the top cell thickening effect. The only physical constrain here would be the presence of sea ice, and indeed the sea ice edge seem border the location where MLD increases (white line & orange blob, figs below). In these higher latitudes heat transformation now happens over thicker cells and with stronger heat loss, which could (possibly) convect more and denser waters locally. 
In turn, In the Southern Hemisphere, the Antarctic continent prohibits convection to move southwards where heat loss would be stronger. Add to that: SWMT on OM2 and MOM6 Antarctic Shelf is driven by salt fluxes, so heat loss wouldn’t be a constraining factor anyways?
![NADeepConvection_MLDchange_SIedge](https://github.com/willaguiar/DSW-collaborative-project/assets/70033934/0568a940-9e9b-463a-831e-db1654c305a0)

**Comment by dkhutch** [comment]
Very nice to see how you've lined up the MLD field with where the changes are occurring. It does seem to make sense that the DSW is mainly shifting northwards, and NADW is shifting towards heavier density classes, while its overall magnitude remains similar.

**Comment by adele-morrison** [comment]
I don’t quite understand what the argument is for the hemispheric difference not being due to the salt vs heat driven formation? To me that still seems plausible. 

In the Antarctic, the formation is all due to sea ice formation, which extracts fresh water out of the top cell. When that top cell is thin, it gets super salty and convects more. This brings up more heat from below, which drives stronger heat loss also.

Whereas in the Atlantic, the formation doesn’t seem to have any influence from sea ice - the salt fluxes are actually making the waters lighter not denser. But there is sufficiently large heat loss to the atmosphere to form dense water. If (and this bit I’m not sure about) the thicker top cell could result in more heat loss, then that could explain the larger North Atlantic formation at high densities.  

Possibly this effect of stronger heat loss with a thicker top cell also occurs in the Antarctic, but it’s overwhelmed by the salinity response and subsequent change in vertical connective heat flux from below, so can’t be seen.

**Comment by willaguiar** [comment]
Oh - that makes sense. I didn't consider that salt-driven convection would also alter the heat exchange at surface. Then comparing heat WMT in a salt-driven convection location with a heat WMT in a heat-driven convection location isn't a clean comparison then. Thanks for the explanation

**Comment by wghuneke** [comment]
I agree with Adele, the SWMT being dominated by salt (Antarctic) and heat (Arctic) can help in understanding what's going on. The way I think about it currently is the following: 

(i) In the Antarctic, SWMT is dominated by salt and mechanical forcing (wind) is mostly responsible for creating an ice-free surface ocean. Brine rejection during sea ice formation is sufficient to increase density and it's easier for the model to convect for a shallow upper cell. Because the winds create polynyas, there's no (less) need for heat to be brought to the surface to keep the process going. 

(ii) In the Arctic, it's more about open ocean convection. In order to maintain an ice-free surface, the model relies on warm waters to be brought up to the surface. That's much easier with thicker grid cells which will have a larger heat capacity and favour heat loss to the colder atmosphere. 

Does this make sense to others?

**Comment by AndyHoggANU** [comment]
Thanks @willaguiar for the detailed analysis, and sorry to be slow to respond - been away the last few days. Yes @wghuneke, your explanation does make sense, but I'm not convinced that it explains everything. In particular, I am wondering why the heat capacity of the top cell is so important when these regions are convecting meaning that there should be rapid vertical heat transport within the mixed layer -- and the mixed layer is much deeper than just the top cell.

Perhaps we need to consider the speed at which KPP can act to "convect" (in quotes, because it doesn't really convect, it actually just enhances mixing). Let's assume that a thinner top cell acts to slow down the rate at which "convection" can transport density downwards - in effect, the dense water has to get through more layers. Then, in the Antarctic, a thinner top cell means that the dense water stays at the surface for longer, so that it can continue to get denser (relative to the thick top cell case). [Note that the most effective way to inject APE into a convection system is to make the densest waters denser ...]

In the Lab Sea, perhaps a thinner top cell means that "convection" is slowed, which means colder water stays at the surface for longer and thereby reduces the heat exchange with the atmosphere. This creates the opposite effect to the salt-driven convection case. 

So, the hypothesis is that the effect could be explained "rate" of "convection" being dependent on the number of layers, in combination with the differing feedbacks of heat-driven SWMT vs salt-driven SWMT.

Is this plausible? If yes, I think we can actually devise a simple 1-D test (perhaps in MOM6?) which looks at the dependence of the vertical evolution of heat and salt upon the number of levels ...

**Comment by willaguiar** [comment]
Hi Andy

It makes sense - if I understand it correctly (thinner surface cells lasting longer in Antarctic Shelf surface would get denser because of continuous salt input, but in the case of heat loss, longer surface residence diminish the rate at which warm water comes to the surface, decreasing heat loss from ocean ).  is that it?


On another note.... If a difference in kpp "convection rate" with resolution is what is causing the contrasting behavior... would that show up in the in the vertical diffusivity diag? I saved previously `diff_cbt_conv` (_vert diffusivity from kpp convection_), and could check how it differs between experiments.

**Comment by AndyHoggANU** [comment]
I'm not sure that the diffusivity will capture it -- maybe the diffusive flux would? But then again, the diffusive flux might be slaved to the surface flux, so may not show anything. 

I think we might need additional (smart but small) simulations to tease this apart...
